### ISSUE TITLE
Login issue: probably related to guild

### ISSUE THREAD

**Issue by AschenStrom** [WarEmu/WarBugs#10419]
Hi everyone,
it's a couple of days that my main character has a problem with logins and teleports. Logging in always takes 3-4 tries before working (and i often manage to log in into a completly black area, named Zone 0, where i have to tp away to leave it).

![narzug_000](https://user-images.githubusercontent.com/31918990/30368330-22022bd6-9871-11e7-9151-d1f0563ce671.jpg)

Exactly like in the "technical support sticky": I can reach the character selection screen, and I see my characters there, but when I attempt to enter the game, I'm kicked out after the loading screen.
The strange thing is that the other lvl 1 character can login without any problem.
The main charactes get also disconnected nearly everytime i teleport (using the book or taking a fly).
Reinstalling the game doesn't seem to fix the problem.

After some tests, it seems related in some way to the guild:
in facts, the problems started a couple of days ago when i joined the guild and, if i quit the guild, everything seems to work fine (while rejoining it brings back all the issues).

I know that 300+ members guilds could have some kind of problems, but this one has 210 members.
Actualy 2 peoples have these problems in the same guild.

Byebye
Narzug BO 39

**Comment by GentleN** [comment]
I also suffered from this, and it has been for almost 3 months. 
Tried to report this contless times but no answer. 
Untill recently i figured it was the guild causing the issue and shared with my guild ( due to wanted to leave because of that. - Narzug and i were from the same guild, Da Freebooterz.
P.S - My goblin went to the guild, and instantly got crashes on loading... then left the guild in curiosity and poof, no more kick offs o_o Magic!!!)
Now i can do scenarios/explore without getting kicked off the loading with my main BO again \o/
Hope this helps and may this issue be fixed in the future (if possible), so that we can be part of a guild again, safely.

Girgur Tuffjaw -BO 40

**Comment by GentleN** [comment]
I also suffered from this, and it has been for almost 3 months. 
Tried to report this contless times but no answer. ( for example  #10084 )
Untill recently i figured it was the guild causing the issue and shared with my guild ( due to wanted to leave because of that. - Narzug and i were from the same guild, Da Freebooterz.
P.S - My goblin went to the guild, and instantly got crashes on loading... then left the guild in curiosity and poof, no more kick offs o_o Magic!!!)
Now i can do scenarios/explore without getting kicked off the loading with my main BO again \o/
Hope this helps and may this issue be fixed in the future (if possible), so that we can be part of a guild again, safely.

Girgur Tuffjaw -BO 40

**Comment by Azorast** [comment]
Same problem here. Endless loading screens and the kicked to the main menu and sometimes getting this screen:

![war 2017-09-30 12-47-57](https://user-images.githubusercontent.com/32238699/31045177-20785716-a5de-11e7-8c3d-c79840220465.png)

I tried deleting "user", "interface" folders but the same problem with this main char. The alter low char (without guild) logging its ok. 

P.D: casually my character belongs to the same guild!: Da Freebooterz

**Comment by Grokmar** [comment]
Hello there, same problem with my Black Orc who is in the Freebooterz guild too, the first time the game log me out when I tried to teleport to Badlands with the flymaster, then I tried many times to type "/stuck" twice, once, whatever, it doesn't work. But the loading was not in the same zone each time: after Badland he sent me to the Capital, where it's supposed to work, then to Norsca. Still loggin' out and can't reach the game. 
Hope the problem will disappear, thanks and...  Waaagh !
